charts/cert-manager/v0.9.1/app-readme.md

```diff
@@ -27,3 +27,8 @@ spec:
     - myingress.com
     secretName: myingress-cert # < cert-manager will store the created certificate in this secret.
 ```
+
+#### Upgrade Notes:
+Upgrading cert-manger from previous version v0.5.2 to v0.9.1 is only supported with app deployed to the kube-system namespace.
+
+User can creating a new cert-manager app in the kube-system namespace and deleting the old version if you have old cert-manager deployed in another namespace.
```

Comment:
If your previous cert-manager application is deployed in a different namespace, you create a new cert-manager application in the `kube-system` namespace and delete the previous version.

Reply:
Let's move the upgrade notes to the top of the app readme.